appendix-basic65.tex

```diff
@@ -459,7 +459,11 @@ \subsection{BACKGROUND}
 \item [Format:] {\bf BACKGROUND colour}
 \item [Usage:] Sets the background colour
                of the screen to the argument, which must be in the
-               range of 0 to 15. (See colour table).
+               range of 0 to 255. All colours within this range are
+               customisable via the {\bf PALETTE} command, though on
+               startup, the ROM provides the first 32 preset colours
+               described in the colour table below.
+
```

Comment:
I'd change the wording slightly. 

```... via the PALETTE command. On startup, the MEGA65 only has the first 32 colours configured, which are described in the table below.```

Do the colours go back to the default 32 on reset as well?

Reply:
It's probably interchangeable but I prefer `only has` over `has only`

Reply:
Ok, switching over